static/templates/tab_bar.hbs

```diff
@@ -1,19 +1,19 @@
-<ul id="tab_list">
-    {{#each tabs}}
-    <li class="{{active}} {{cls}}" {{#if data}}data-name="{{data}}"{{/if}}>
+<span id="tab_list">
+    <span {{#if extra_class}}class="{{extra_class}}"{{/if}} {{#if data}}data-name="{{data}}"{{/if}}>
         {{#if icon}}
-        <i class="fa fa-angle-right" aria-hidden="true"></i>
+        <i class="fa fa-{{icon}}" aria-hidden="true"></i>
         {{/if}}
-        {{! Most tabs are links, but some are not since we don't have a narrow for them (e.g. Search) }}
-        {{#if hash}}
-            <a href="{{hash}}">{{title}}</a>
+        {{#if stream_settings_link}}
+        <a class="settings_link" href="{{stream_settings_link}}">{{title}}</a>
         {{else}}
-            {{#if home}}
-            <i class="fa fa-home" aria-hidden="true"></i>
-            {{else}}
-            {{title}}
-            {{/if}}
+        {{title}}
         {{/if}}
-    </li>
-    {{/each}}
-</ul>
+    </span>
+    {{#if sub_count}}
+    <span class="sub_count" data-toggle="tooltip" title="{{sub_count}} users are subscribed to #{{title}}"><i class="fa fa-user-o"></i>{{formatted_sub_count}}</span>
+    {{/if}}
+    {{#if rendered_narrow_description}}
+    <span class="narrow_description" data-toggle="tooltip" title="{{rendered_narrow_description}}">{{{rendered_narrow_description}}}</span>
```

Comment:
Use the `{{ rendered_markdown rendered_narrow_description}}` handlebars helper for this, rather than `{{{`.